### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by michelleegan11**
## PR Description

Added missing landing pages to SCOS developer guides

## Checklist
- [x ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/scos/dev/best-practices/search-best-practices/search-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+[Usage-driven schema and document structure](/docs/scos/dev/best-practices/search-best-practices/usage-driven-schema-and-document-structure.html)
+
+[Multi-term auto completion](/docs/scos/dev/best-practices/search-best-practices/multi-term-auto-completion.html)
+
+[Naive product centric approach](/docs/scos/dev/best-practices/search-best-practices/naive-product-centric-approach.html)
+
+[On-site search](/docs/scos/dev/best-practices/search-best-practices/on-site-search.html)
+
+[Personalization - dynamic pricing](/docs/scos/dev/best-practices/search-best-practices/personalization-dynamic-pricing.html)
```
[the remark this answers is not in the document]
Comment (michelleegan11) [reply docs/scos/dev/best-practices/search-best-practices/search-best-practices.md]: I added bullets. Please just let me know if you need me to do anything else with this page

File: docs/scos/dev/best-practices/state-machine-cookbook/state-machine-cookbook.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    link: docs/scos/dev/best-practices/state-machine-cookbook/state-machine-cookbook-build-a-state-machine.html
+---
+
+The Spryker State Machine allows you to easily implement complex processes and map out business workflows. Instead of writing code for every process or adapting standard functionality, Spryker State Machine lets you model the order process with states and transitions.
+
+For example, you can define one state machine to manage orders that use a credit card payment type, and a separate state machine for orders that use invoices. 
+
+A feature heavily utilized by Spryker customers, a state machine translates a variety of business needs into a coherent IT structure.
+
+This section explains state machine fundamentals then explains how to model a state machine.
```
Comment (michelleegan11) [inline docs/scos/dev/best-practices/state-machine-cookbook/state-machine-cookbook.md:18]:
```suggestion
This section explains state machine fundamentals and how to model a state machine. 
```

Comment (michelleegan11) [reply docs/scos/dev/best-practices/state-machine-cookbook/state-machine-cookbook.md:18]: done

File: docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/glue-backend-api.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    link: docs/scos/dev/glue-api-guides/202204.0/decoupled-glue-infrastructure/how-to-guides/how-to-create-or-change-a-convention.html
+  - title: How to create grant type parameters
+    link: docs/scos/dev/glue-api-guides/202204.0/decoupled-glue-infrastructure/how-to-guides/how-to-create-grant-type-parameters.html
+  - title: How to use an authentication server
+    link: docs/scos/dev/glue-api-guides/202204.0/decoupled-glue-infrastructure/how-to-guides/how-to-use-an-authentication-server.html
+  - title: Authentication and authorization
+    link: docs/scos/dev/glue-api-guides/202204.0/decoupled-glue-infrastructure/authentication-and-authorization.html
+  - title: Security and authentication
+    link: docs/scos/dev/glue-api-guides/202204.0/decoupled-glue-infrastructure/security-and-authentication.html
+---
```
[the remark this answers is not in the document]
Comment (michelleegan11) [reply docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/glue-backend-api.md:40]: done!

**Review by lenadoc — APPROVED** [review]

**Pull request merged**